studio-ui/static-assets/themes/cstudioTheme/yui/assets/rte.css

```
@@ -0,0 +1,95 @@
+table.cstudioTableLayout caption,
+table.cstudioTableLayout tbody,
+table.cstudioTableLayout tfoot,
+table.cstudioTableLayout thead,
+table.cstudioTableLayout tr,
+table.cstudioTableLayout th,
+table.cstudioTableLayout td {
+	border: 1px solid #ccc !important;
+}
+/** End of Table Layout styles **/
+
+/** Right-Click property box for RTE**/
```

Stylelint (17.14.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)